### PR TITLE
Replace `768px` breakpoint Sass variables with tokens

### PR DESCRIPTION
**Pull request by lgriffee**
### WHY are these changes introduced?
Beginning of work to consolidate and standardize all of our breakpoint values across `polaris` and `web`.

Related to #5620 

### WHAT is this pull request doing?
Replacing any `768px` Sass breakpoint variables in Polaris' components with the new `p-breakpoints-md` token.

### CODE REVIEW

File: polaris-react/UNRELEASED.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ Use [the changelog guidelines](/documentation/Versioning%20and%20changelog.md) t
 - Added `border-width-4` and `border-width-5` tokens and replaced hardcoded values ([#5528](https://github.com/Shopify/polaris/pull/5528))
 - Replaced any hardcoded `outline-width` with `border-width` ([#5528](https://github.com/Shopify/polaris/pull/5528))
 - Added the ability to disable specific dates in the `DatePicker`, to go along with date ranges ([#5356](https://github.com/Shopify/polaris/pull/5356))
-
 - Added breakpoint CSS custom properties and SCSS media conditions ([#5558](https://github.com/Shopify/polaris/pull/5558))
+- Replaced `768px` breakpoint Sass variables with tokens ([#5629](https://github.com/Shopify/polaris/pull/5629))
```
Comment (aaronccasanova) [inline polaris-react/UNRELEASED.md]:
```suggestion
- Replaced `768px` breakpoint mixins and variables with custom media conditions ([#5629](https://github.com/Shopify/polaris/pull/5629))
```

**Review by aaronccasanova — APPROVED** [review]

**Comment by zecarlostorre** [comment]
@lgriffee 

Screenshot 2022-04-27 at 18 54 06

In this instance where we have 6px (0.375em), we can use --p-space-2 (8px = 0.5em) - I double checked with the designer that worked on it (@ouellettejordan) to make sure

File: polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss [hunk 2]
```diff
@@ -1,6 +1,5 @@
 @import '../../../../styles/common';
-
-$breakpoint: 768px;
+@import '../../../../styles/media-queries';
```
Comment (alex-page) [inline polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss]: Should this just be added to `common` so we don't need to import it everywhere?

Comment (aaronccasanova) [reply polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss]: Works for me! Think we just need to update the output directory in `gen-media-queries.js` and add it to `src/styles/common` 👍

Comment (lgriffee) [reply polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss]: We ended up adding the export to `common`!

**Pull request merged**